DREAM3DReviewFilters/util/EBSDWriterFactory.cpp

```diff
@@ -0,0 +1,143 @@
+#include "EBSDWriterFactory.h"
+
+#include <cmath>
+#include <fstream>
+
+#include "SIMPLib/DataArrays/DataArray.hpp"
+#include "SIMPLib/DataArrays/StringDataArray.h"
+
+namespace EBSDWriterHelper
+{
+
+template <typename T>
+inline QString WriteArrayToQStringLine(QString rowData, const IDataArray::Pointer& ptr, uint64_t index)
+{
```

Comment:
rowData is passed by copy but then appended to and copied again for the return. We could pass in by reference and not return anything. Thoughts?